py-polars/polars/internals/expr/expr.py

```diff
@@ -5405,6 +5405,23 @@ def shuffle(self, seed: int | None = None) -> Expr:
         │ 3   │
         └─────┘
 
+        Examples
+        --------
+        >>> df = pl.DataFrame({"a": [1, 2, 3]})
+        >>> df.select(pl.col("a").shuffle(seed=1))
+        shape: (3, 1)
+        ┌─────┐
+        │ a   │
+        │ --- │
+        │ i64 │
+        ╞═════╡
+        │ 2   │
+        ├╌╌╌╌╌┤
+        │ 1   │
+        ├╌╌╌╌╌┤
+        │ 3   │
+        └─────┘
```

Comment:
I think you duplicated this examples section by mistake?

Reply:
@stinodego No, I think this crept in during a merge with master. Thought I'd found all of them, let me know if you find any more

Reply:
Removed now